pyuvdata/uvdata.py

```diff
@@ -450,6 +458,36 @@ def antnums_to_baseline(self, ant1, ant2, attempt256=False):
                 warnings.warn(message)
 
         return np.int64(2048 * (ant1 + 1) + (ant2 + 1) + 2**16)
+    
+    def order_pols(self,order='AIPS'):
```

Comment:
I'm afraid to ask if there is a test covering this scary thing. There should be.